run-test.cmd

```diff
@@ -1,16 +1,24 @@
 @if not defined _echo @echo off
-
+setlocal 
 :: To run tests outside of MSBuild.exe
-:: %1 is the path to the tests\<OSConfig> folder
-:: %2 is the path to the packages folder
+:: %1 is the path to the bin\<OSConfig> folder
+:: %2 is the path to the tools\testdotnetcli folder
+
+set LOCATION=%1
+set RUNTIME_PATH=%2
+
+if "%LOCATION%" == "" set LOCATION=%~dp0\bin\Windows_NT.AnyCPU.Debug
+if "%RUNTIME_PATH%" == "" set RUNTIME_PATH=%~dp0\Tools\testdotnetcli
```

Comment:
This is broken again after a recent change. We really should avoid having these custom run-test.sh/cmd files which aren't linked with the rest of our build logic. We should instead run them via msbuild so the logic is shared.